packages/block-editor/src/components/block-styles/preview.native.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import {
+	View,
+	TouchableWithoutFeedback,
+	Text,
+	Dimensions,
+	Animated,
+	Easing,
+	Image,
+} from 'react-native';
+/**
+ * WordPress dependencies
+ */
+import { BottomSheet } from '@wordpress/components';
+import { useState, useEffect, useRef } from '@wordpress/element';
+import { usePreferredColorSchemeStyle } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import styles from './style.scss';
+
+const MAX_ITEM_WIDTH = 120;
+
+function StylePreview( { onPress, isActive, style, url } ) {
+	const [ itemWidth, setItemWidth ] = useState( MAX_ITEM_WIDTH );
+	const { label, name } = style;
+	const opacity = useRef( new Animated.Value( 1 ) ).current;
+
+	function onLayout() {
+		const columnsNum =
+			Math.floor( BottomSheet.getWidth() / MAX_ITEM_WIDTH ) + 0.5;
```

Comment:
I'm guessing `0.5` is some sort of margin?

Reply:
Nope, I'm calculating there a number of columns which should be displayed and we decided to always display `x + 0.5` to indicate there is a scroll availability.

Reply:
Good to know, can we please put it in a variable or have some sort of comment? So others can know what that number does exactly.